interface/src/app/utils.spec.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * Unit tests.
+ */
+
+// Test the hyperGeometricTest function
+describe('Test for hypergeometric test function in utils.js', function() {
+  beforeEach(module('adage.utils'));
+
+  var mockService;
```

Comment:
Ahh, I see, so it is not actually a real mocked service. Ok, I will change it to that, thanks!